src/pages/AddItem.js

```diff
@@ -1,3 +1,61 @@
+import { useState } from 'react';
+import { db } from '../lib/firebase';
+// import { useCollection } from 'react-firebase-hooks/firestore';
+
 export default function AddItem(props) {
-  return <h1>This is the form</h1>;
+  const [itemName, setItemName] = useState('');
+  const [purchaseFrequency, setPurchaseFrequency] = useState(0);
+  const [lastPurchased] = useState(null);
+
+  const handleName = (e) => {
```

Comment:
Naming is _super_ hard but can improve legibility of code for your colleagues and future readers. An old adage goes something like "code is written once but is read many times more". In the interest of helping the reader/reviewer/future-bug-fighter we want to name functions so we understand what they're doing. Understanding _what_, _when_, and _why_ in a name can help clarity.

In the case of `handleName` and `handleFrequency` we may want to ask the question 'When do we handle the name?'. In this case the answer is 'when it changes!' because we are using these functions for `<input onChange={} /> callbacks so `handleNameChange` and `handleFrequencyChange` would be good names!